guides/common/modules/proc_importing-suse-products.adoc

```diff
@@ -22,3 +23,41 @@ By default, _debug_ and _source_ repositories are excluded.
 +
 Note that the SCC Manager plug-in can only add, but not remove products from {Project}.
 . Optional: Click on the task ID to follow the creation of the selected products in {Project}.
+
+[id="cli-Importing_SUSE_Products_{context}"]
+.CLI procedure
+. List all available SUSE products:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer scc_manager scc_accounts scc_products list \
+--location-id _My_Location_ID_ \
```

Comment:
As the scc_account is bound to a location/organization already and the scc_account_id is set, location / organization is not necessary to be set. These options will be removed soon-ish.

Reply:
Applied.